run.sh

```diff
@@ -97,7 +97,33 @@ do
         echo "No images need to be cleaned"
     fi
 
-    rm -f ToBeCleanedImageIdList ContainerImageIdList ToBeCleaned ImageIdList KeepImageIdList
+    # Remove unused volumes
+    echo "=> Start to clean unused volums"
```

Comment:
Could we clean volumes only if they are unused for, lets say, more than 24h?
When the user is redeploying a container, its volumes are "unused" from the time the previous container is destroyed until the time the new container is destroyed. Also, if the new container deployment fails, the volumes would be reused if the user retries a redeploy operation after a while. I think 24h is a good compromise.